src/source/Ice/IceAgent.c

```diff
@@ -1047,6 +1055,9 @@ STATUS iceAgentSendSrflxCandidateRequest(PIceAgent pIceAgent)
             switch(pCandidate->iceCandidateType) {
                 case ICE_CANDIDATE_TYPE_SERVER_REFLEXIVE:
                     pIceServer = &(pIceAgent->iceServers[pCandidate->iceServerIndex]);
+                    if (pIceServer->ipAddress.family != pCandidate->ipAddress.family) {
+                      break;
```

Comment:
We are trying to minimize the usage of the control flow keywords such as break and continue. It's not a good coding style to use break within a switch all together. Use the negation of the boolean predicate here

if (family == other) {
  // do the stuff
}

Reply:
Got it. I'll update it.